marketplace_builder/views/pages/how-to/modules/creating-module.liquid

```diff
@@ -0,0 +1,192 @@
+---
+converter: markdown
+metadata:
+  title: Creating a Module
+  description:  
+slug: tutorials/modules/creating-module
+searchable: true
+---
+
+This guide will help you learn how to create a module by walking you through the steps of creating a simple module. 
+
+## Requirements
+
+This is an advanced tutorial. To follow it, you should be familiar with basic PlatformOS concepts, the topics in the Get Started section, and modules. 
+
+* [How PlatformOS Works](/how-platformos-works)
+* [Get Started](/get-started)
+* [Modules](/tutorials/modules)
+
+## Steps
+
+Creating a module is a three-step process:
+
+1. Create directories
+2. Create GraphQL queries
+3. Create page
+
+
+### Step 1: Create directories
+
+Create two directories inside the `modules` directory called `admincms`, then create the `public` and `private` folders inside it. Place the `graph_queries` directory inside `private`, and the `views/pages` directories inside public. 
+
+See the structural overview of the module you'll create here for reference: 
+
+```
+admincms
+├── private
+│   └── graph_queries
+│       ├── get_customizations.graphql
+│       └── get_pages.graphql
+└── public
+    └── views
+        └── pages
+            └── admin.html.liquid
+```
+
+### Step 2: Create GraphQL queries
+
+Create a GraphQL query that pulls a list of Customizations,`admincms/private/graph_queries/get_customizations.graphql`:
+
+```graphql
+query get_customizations {
+  customizations(
+    per_page: 20
+    sort: [{ name: "human_name", order: "asc" }]
+  ) {
+    total_entries
+    results {
+      human_name
+      properties
+    }
+  }
+} 
+```
+
+Create another GraphQL query that pulls a list of pages, `admincms/private/graph_queries/get_pages.graphql`:
+
+```graphql
+query get_pages {
+    pages(per_page: 100) {
+    total_entries
+    results {
+      format
+      slug
+    }
+  }
+}
+```
+
+### Step 3: Create page
+
+Create the page `admincms/public/views/pages/admin.html.liquid`:
+
+```liquid
+{% raw %}
+---
+slug: admincms
+layout_name: 1col
+---
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mini.css/3.0.0/mini-default.min.css">
```

Comment:
also, when moving it to nearme-examples ill remove it as there is already bootstrap loaded for prettier styling